modules/oke/iam.tf

```diff
@@ -41,5 +47,8 @@ resource "oci_identity_policy" "oke_volume_kms" {
   description    = "Policies for block volumes to access kms key"
   name           = var.label_prefix == "none" ? "oke-volume-kms" : "${var.label_prefix}-oke-volume-kms"
   statements     = local.oke_volume_kms_policy_statements
+  
+  freeform_tags = var.freeform_tags
```

Comment:
May i know why this freeform tag is added for iam. It will assign the full map .